static/templates/user_profile_modal.hbs

```diff
@@ -64,16 +64,22 @@
                 </div>
             </div>
 
-            <div class="subscription-group-list">
-                <div class="tabcontent" id="groups-tab">
-                    <table class="user-group-list" data-empty="{{t 'No user group subscriptions.'}}"></table>
+            <div class="subscription-stream-tab">
+                <div class="subscription-stream-list">
+                    <div class="tabcontent" id="streams-tab">
+                        <table class="user-stream-list" data-empty="{{t 'No stream subscriptions.'}}"></table>
+                    </div>
                 </div>
+            </div>
 
-                <div class="tabcontent" id="streams-tab">
-                    <table class="user-stream-list" data-empty="{{t 'No stream subscriptions.'}}"></table>
+            <div class="subscription-group-tab">
+                <p>&nbsp;&nbsp;&nbsp;View and edit user groups <a id="edit_user_group" href="#organization/user-groups-admin">here</a></p>
```

Comment:
This should be tagged for translation.
Also, I'm not sure about wordings may be something like `View user groups settings` would be better but I think @alya can give better ideas for this.

Reply:
Or maybe user-group popover:
![image](https://user-images.githubusercontent.com/63504956/123044896-eec29d00-d417-11eb-89a2-9bcd5f2a2f14.png)